### DISCUSSION TITLE
Overwrite check status of sqitch

### DISCUSSION THREAD

**Discussion by chamini2** [theory/sqitch#843] [Q&A] [answered]
I had to do some modifications in the past and now my `sqitch check` command returns a failed status. 

```
Script signatures diverge at change alter_table_gateway_requests_duration
 Failed one check
```

How can I ask sqitch to "convince itself" we are good beacuse I am sure we are even though the SHAs dont match?

**Comment by theory** [comment, marked as answer]
Revert to the last known good SHA using `--log-only`, then deploy from there using `--log-only`. That will update the logs without executing the scripts. Be very sure that this is correct.

**Comment by chamini2** [comment]
Thanks! Yes I have compared a `pgdump -s` of the real db and a db from scratch with sqitch migrations.
